src/citrine/resources/workflow_executions.py

```diff
@@ -11,23 +11,41 @@
 
 
 class WorkflowExecution(Resource['WorkflowExecution']):
-    """[ALPHA] A Citrine Workflow Execution."""
+    """[ALPHA] A Citrine Workflow Execution.
+
+    Parameters
+    ----------
+    uid: str
+        Unique identifier of the workflow execution
+    project_id: str
+        Unique identifier of the project that contains the workflow execution
+    workflow_id: str
+        Unique identifier of the workflow that was executed
+    version_number: int
+        Integer identifier that increases each time the workflow is executed.  The first execution
```

Comment:
Not blocking: The increasing increment constraint seems to be a data service (or account service?) concern. What requires mithril to be aware of it?

Reply:
mithril?